pkg/apis/extensions/v1alpha1/types_worker.go

```diff
@@ -175,3 +177,5 @@ type MachineDeployment struct {
 	// Maximum is the maximum number for this machine deployment.
 	Maximum int32 `json:"maximum"`
 }
+
+var RollingUpdate gardencorev1beta1.ConditionType = "RollingUpdate"
```

Comment:
Can this be a const? Doc string missing. Should we name it `ConditionRollingUpdate` similar to other condition type constants?

Reply:
Sure will add the doc strings. 
Where do you got the naming from - see e.g [here](https://github.com/gardener/gardener/blob/fc8758d8ebafd9c17b0eeca723495d8796a27e52/pkg/apis/core/v1beta1/types_seed.go#L262) or [here](https://github.com/gardener/gardener/blob/fc8758d8ebafd9c17b0eeca723495d8796a27e52/pkg/apis/core/v1beta1/types_shoot.go#L963-L962).

Reply:
Hm, then maybe let's prefix with the resource kind, i.e. `WorkerRollingUpdate`